### ISSUE TITLE
The user accidentally clicks the block user button, the user can unblock the user there

### ISSUE THREAD

**Issue by chenzisuifeng** [WWBN/AVideo#8061]
The user accidentally clicks the block user button, the user can unblock the user there
![image](https://github.com/WWBN/AVideo/assets/47063171/662e98f5-f96e-46a1-acbe-b93d1d63573a)

**Comment by DanielnetoDotCom** [comment]
If you access the blocked user profile or video you may be able to see the unblock button

**Comment by chenzisuifeng** [comment]
But I can't find the blocked user profile or video

**Comment by DanielnetoDotCom** [comment]
you may need access to his direct channel. if you are the admin you may be able to find out.

**Comment by chenzisuifeng** [comment]
Yes, it is easy for an administrator to find his direct channel, but it is quite troublesome for ordinary users
